code-challenges.test.js

```diff
@@ -37,6 +75,47 @@ const nums2 = [109, 5, 9, 67, 8, 24]
 // b) Create the function that makes the test pass.
 
 
+// I am going to create a test that looks at a function call returnArrayMinMax, it wll return an array containg the minumum number and maximum number
+
+        // describe ("returnArrayMinMax", () => {
+        //     it("takes in an array and returns an array containing the least and max value", () => {
+        //         expect(returnArrayMinMax(nums1)).toEqual([-8, 90])
+        //     })
+        // })
+
+
+// test failed because function not defined, going to create a function now that takes in the array, takes the maximum value and the minimum value and returns a new array containing them
+
+        // const returnArrayMinMax = ( arrayInput) => {
+        //     var largeNum = arrayInput[0]
+        //     var smallNum = arrayInput[0]
+
+        //     for (var i = 1; i < arrayInput.length; i++) {
+
+        //         if (arrayInput[i] > largeNum) {
+        //             largeNum = arrayInput[i]
+        //         } 
+        //     }
+
+        //     for (var i = 1; i < arrayInput.length; i++) {
+
+        //         if (arrayInput[i] < smallNum) {
+        //             smallNum = arrayInput[i]
+        //         } 
+        //     }
+
+        //     returnArray = []
+
+        //     returnArray.push(smallNum)
+        //     returnArray.push(largeNum)
+        //     return returnArray
+
+        // }
```

Comment:
Your code looks good, just lacking adequate pseudocode.  I want to also encourage you if you want to stretch yourself, try refactoring your functions once you get your tests working to simplify a little more.